src/structures/engine.go

```diff
@@ -0,0 +1,40 @@
+package structures
+
+type SEInfo struct {
+	SEDomain   string
+	SEName     string
+	SEURL      string
+	ResPerPage int
+}
```

Comment:
Change names to Domain, Name, and URL, they are already in a struct called SEInfo.